build_tools/circle/build_test_arm.sh

```diff
@@ -21,39 +21,51 @@ source build_tools/shared.sh
 
 sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
 sudo apt-get update
-sudo apt-get install python3-virtualenv ccache
-python3 -m virtualenv --system-site-packages --python=python3 testenv
-source testenv/bin/activate
-pip install --upgrade pip
+
+# Setup conda environment
+MINICONDA_URL="https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh"
+
+# Install Mambaforge
+wget $MINICONDA_URL -O mambaforge.sh
+MINICONDA_PATH=$HOME/miniconda
+chmod +x mambaforge.sh && ./mambaforge.sh -b -p $MINICONDA_PATH
+export PATH=$MINICONDA_PATH/bin:$PATH
+mamba update --yes conda
+
+# Create environment and install dependencies
+mamba create -n testenv --yes python=3.7
+source activate testenv
```

Comment:
```suggestion
mamba activate testenv
```

Reply:
Not sure this will work when we install mamba in a CI environment with the `-b` flag and without restarting a new shell session.

Reply:
ah right. I assume that only the path needs to be added but let the `source` as-is